Big-Spender/readme.md

```diff
@@ -67,24 +72,38 @@ INSERT YOUR QUERY HERE
 
 **You:** Then here's the query for that:
 
-```sql
+<!-- ```select * from spends where description ilike '%FeE%' -->
+
 INSERT YOUR QUERY HERE
-```
+
+````
+                select * from spends where description ilike '%FeE%'
```

Comment:
Good work taking capitalisations into consideration with 'ilike'. note you don't have to mix cases in the word, e.g '%FeE%' can simply be written as '%fee%'. ILIKE already handles the case insensitive matching